content/en/tests/flaky_health/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Test Health
+description: Understanding and Quantifying the impact of your tests
+further_reading:
+- link: "/continuous_integration/tests/"
+  tag: "Documentation"
+  text: "Learn about Test Optimization"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Test Optimization is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Overview
+
+Effective test optimization involves not only identifying flaky tests but also providing insights that help Developer Experience teams (such as DevOps or Platform teams) clearly demonstrate value and manage their CI pipelines and test sessions more efficiently. The impact of failing jobs extends across multiple dimensions, including increased infrastructure costs from rerunning test jobs, decreased deployment frequency, and lost developer focus time.
+
+Test Health provides insights at the repository and test service levels, focusing on pipelines that failed due to flaky tests, time wasted in CI due to flaky tests, pipelines prevented from failing by Test Optimization, and time saved in CI thanks to the Test Optimization product.
+
+## Key Metrics
+
+### Pipelines Failed
+- **Failures due to non-flaky tests**: Count of CI pipelines with failed test sessions containing non-flaky test failures.
+- **Failures due to flaky tests**: Count of CI pipelines with failed test sessions where all test failures are flaky (`@test.is_known_flaky` or `@test.is_new_flaky`).
+- **Percentage of failures due to flaky tests**: `(Flaky Failures / Total Failures)`.
+
+### Time Wasted in CI
+- **Lost duration due to failed tests**: Total duration of failed test sessions containing test failures.
+- **Lost duration due to flaky tests**: Total duration of failed test sessions where all test failures are flaky (`@test.is_known_flaky` or `@test.is_new_flaky`).
+
+### Pipelines Saved
+
+- **Number of pipelines saved due to auto-retries**: Number of CI pipelines with passed test sessions containing tests with `@test.is_retry:true` and `@test.is_new:false`.
+- **Percentage of pipelines saved due to auto-retries**: `(Number of CI pipelines saved due to auto-retries / Total number of CI pipelines with tests)`.
+
+### Time Saved in CI
+- **Saved time due to auto-retries**: Total duration of passed test sessions containing tests with `@test.is_retry:true` and `@test.is_new:false`. Individual test retries will have a minor impact on total duration saved.
```

Comment:
Now I'm thinking about it, it would be better to say:

Total duration of passed test sessions in which some tests initially failed but later passed due to the [Auto Test Retry](https://docs.datadoghq.com/tests/flaky_test_management/auto_test_retries) feature.